hudi-common/src/main/java/org/apache/hudi/common/fs/FSUtils.java

```diff
@@ -415,17 +420,18 @@ public static boolean isLogFile(Path logPath) {
     return matcher.find() && logPath.getName().contains(".log");
   }
 
+  public static boolean isDataFile(Path path) {
```

Comment:
this can be ba method of `HoodieFileFormat`, rather than a util method here.

Reply:
Done